frontend/summary/summary/RoBHeatmapPlot.js

```diff
@@ -452,13 +451,36 @@ class RoBHeatmapPlot extends D3Visualization {
     }
 
     build_legend() {
-        if (this.legend || !this.data.settings.show_legend) return;
-        let rob_response_values = this.data.aggregation.studies[0].data.rob_response_values,
+        if (this.legend || !this.data.settings.show_legend) {
+            return;
+        }
+        const rob_response_values = this.data.aggregation.studies[0].data.rob_response_values,
             options = {
                 dev: this.options.dev || false,
                 collapseNR: false,
-            };
-        this.legend = new RoBLegend(this.svg, this.data.settings, rob_response_values, options);
+            },
+            getFootnoteOptions = () => {
+                let footnotes = [];
+                if (this.cells_data.filter(d => d.robArray.length > 1).length > 0) {
```

Comment:
The asterisk is only applied if the number of unique scores is > 1, so there can be a mismatch of legend to visual if there are multiple scores but they're the same score. I have included a screenshot of an example where it was rescored and the score stayed the same but bias changed.
![image](https://user-images.githubusercontent.com/46504563/82357816-9ed30480-99d3-11ea-95ab-8e353968fe1f.png)

So I suppose I would suggest chaining a uniq() to the end of the filter to be in line with how the asterisk is actually decided. This is a very specific use case and probably won't happen in actual practice, though.
For reference, [where the asterisk is added](https://github.com/shapiromatron/hawc/blob/e996a17a2a3debe20dd3eb42caef0b6d6cd1a50b/frontend/riskofbias/constants.js#L121)

Reply:
Fixed.